addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3169,8 +3169,8 @@ var SnippetsMenu = Widget.extend({
         const docSelection = document.getSelection();
         const $currentSelectionTarget = docSelection && docSelection.rangeCount > 0 ? $(docSelection.getRangeAt(0).commonAncestorContainer) : $();
         // Do not  toggle visibility if the target is inside the toolbar ( eg. during link edition).
-        if ($currentSelectionTarget.parents('#toolbar').length ||
-            (e && $(e.target).parents('#toolbar').length)
+        if ($currentSelectionTarget.parent().closest('#o_we_editor_toolbar_container').length ||
+            (e && $(e.target).closest('#o_we_editor_toolbar_container').length)
```

Comment:
Because $currentSelectionTarget is a selection so It will be never the toolbar container,

Reply:
can we not rather ping someone on this PR?

Reply:
Does not change the fact it should be in its own commit so I can then ask them to review the commit in this PR

Reply:
> Because $currentSelectionTarget is a selection so It will be never the toolbar container,

If the selection can be inside a child of the container, why can't it be inside the container itself? :thinking:

Reply:
ok, I remove the parent () to be sure

Reply:
yes It could be in the containe but not the container itself.